src/utils/flow.js

```diff
@@ -1,16 +1,25 @@
 /* @flow */
 import type { FlowType, FlowTypes } from "./types";
 
-function flowTypeAnnotationToString(type: TypeTypeAnnotation, nullable: boolean): FlowType {
+function flowTypeAnnotationToString(type: TypeTypeAnnotation, nullable: boolean, variables?: Object): FlowType {
   switch (type.type) {
     case "BooleanTypeAnnotation":
-      return { type: "boolean", nullable };
+      return { type: "boolean", nullable, variables };
     case "NumberTypeAnnotation":
-      return { type: "number", nullable };
+      return { type: "number", nullable, variables };
     case "StringTypeAnnotation":
-      return { type: "string", nullable };
+      return { type: "string", nullable, variables };
+    case "StringLiteralTypeAnnotation":
+    case "BooleanLiteralTypeAnnotation":
+    case "NumericLiteralTypeAnnotation":
+      return {
+        type: "literal",
+        nullable,
+        variables,
+        value: typeof type.value === "string" ? type.value.replace(/'/g, "\"") : type.value
+      };
```

Comment:
Cool, no point doing it if the relay plugin will handle it. Is it easy to test that in expected combined?